### PR TITLE
Feature 3.6.3/vpack exception backtrace logger

### PR DESCRIPTION
**Pull request by maierlars**

### CODE REVIEW

File: arangod/Aql/RestAqlHandler.cpp [hunk 1]
```diff
@@ -464,9 +464,13 @@ RestStatus RestAqlHandler::useQuery(std::string const& operation, std::string co
   }
 
   try {
+    VPackSlice::emptyArraySlice().at(0);
```
Comment (maierlars) [inline arangod/Aql/RestAqlHandler.cpp:467]: Will remove this soon.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/9995/

**Comment by KVS85** [comment]
@maierlars can you please address compilation failure on Windows (http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-windows/3844/EDITION=enterprise,STORAGE_ENGINE=rocksdb,TEST_SUITE=single,limit=windows&&test/):
```
C:\vm03-windows\oskar\work\ArangoDB\3rdParty\velocypack\src\Exception.cpp(32): fatal error C1083: Cannot open include file: 'execinfo.h': No such file or directory
```

**Comment by KVS85** [comment]
Closed since libmusl can't handle this implementation.

https://arangodb.atlassian.net/browse/BTS-69 was created for another similar problem but in devel only.

**Pull request closed without merging**